### DISCUSSION TITLE
does unused svelte components make my website slower?

### DISCUSSION THREAD

**Discussion by tryggs** [sveltejs/kit#5611] [Q&A]
if I have imported a component but not used it, does it affect the sites performance?

**Comment by hartwm** [comment]
if by not used, you mean not imported anywhere then no, as svelte is compiled and not virtual dom

**Reply by tryggs** [reply]
but of it is imported, just not rendered?

**Reply by PlopTheReal** [reply]
I may be wrong but if imported then its code should be in the server response. You may not want to import it if not used.

**Reply by tryggs** [reply]
ok thanks
